src/main/java/co/inventorsoft/academy/springresttask/controller/EmailController.java

```diff
@@ -0,0 +1,81 @@
+package co.inventorsoft.academy.springresttask.controller;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.List;
+import javax.validation.Valid;
+
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import co.inventorsoft.academy.springresttask.EmailDAO;
+import co.inventorsoft.academy.springresttask.EmailService;
+import co.inventorsoft.academy.springresttask.EmailsToWeb;
+import co.inventorsoft.academy.springresttask.VerySimpleMail;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+@RestController
+@RequestMapping(value = "/emails")
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@AllArgsConstructor
+public class EmailController {
+	
+	EmailDAO emailDAO;
+	EmailsToWeb emailsToWeb;
+	EmailService emailService;
+
+    @GetMapping(produces = "application/json")
+    public List<VerySimpleMail> getEmails() {
+        return emailsToWeb.convert( emailDAO.getAll() );
+    }
+
+    @PostMapping(value = "/new", consumes = "application/json")
```

Comment:
URL is a name. You will not call you dog new, right? The same goes to url: it should be a noun. Not a verb! Use request method to explain what to do. Example: POST /emails - means that you post(or add, to simplify) something to collection of emails. Here is the good explanation of how to create good urls https://www.restapitutorial.com/lessons/restfulresourcenaming.html